bundler/spec/support/helpers.rb

```diff
@@ -192,12 +192,18 @@ def sys_exec(cmd, options = {})
         command_execution.exitstatus = wait_thr && wait_thr.value.exitstatus
       end
 
-      (@command_executions ||= []) << command_execution
-
       unless options[:raise_on_error] == false || command_execution.success?
-        raise "Invoking #{cmd} failed!"
+        raise <<~ERROR
+          Invoking #{cmd} failed!
```

Comment:
Not particularly related to this, but I would expect this `#{cmd}` to expand the actual `cmd`.
For instance, here: https://github.com/rubygems/rubygems/runs/791381071#step:10:280

Instead of throwing `raise "Invoking #{cmd} failed!"`, it should rather behave something like:
https://github.com/rubygems/rubygems/runs/791381071#step:10:283

Is it a feature? Or a bug? :sweat_smile:

Reply:
First link is telling you the exact line of code where the failure happened, so it's not expanding it. Second it's showing the actually error raised, so it's being expanded. I believe it's a feature :)

Reply:
Okay, okay. Fair enough, thanks! :smile: